File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int [] arr;
    public int[] getConcatenation(int[] nums) {
        arr = new int[nums.length*2];
        int j = 0;
        for(int i = 0 ; i < nums.length;i++)
        {
            arr[j] = nums[i];
            j++;
        }
        for(int i = 0 ; i < nums.length ; i++)
        {
            arr[j] = nums[i];
            j++;
        }
        return arr;    
    }
}

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.cpp
#include <algorithm>
using namespace std;

class Solution {
public:
    vector<int> replaceElements(vector<int>& arr) {
       vector<int> vResult;

        for(int i = 0 ; i < arr.size() ; i++)
        {
            if(i == arr.size() - 1)
            {
                vResult.push_back(-1);
            }
            else
            {
               auto itr = max_element(arr.begin() + i + 1 , arr.end());

               vResult.push_back(*itr); 
            }


        }

        return vResult;
    }
};